client/src/helpers/functions/usePagination.ts

```diff
@@ -16,7 +16,7 @@ interface UsePaginationProps {
   currentPage: number;
 }
 
-export const usePagination = ({
+const usePagination = ({
   totalCount,
   pageSize,
   siblingCount = 3,
```

Comment:
what is sibling count? is it the previous page? if so, why is it set to 3? provide a comment here about that